Add PT-BR automatic translation pkg_ci.pt.Rmd (#824)
ropensci/dev_guide: pkg_ci.pt.Rmd

#### Testes usando diferentes versões do R {#testing-using-different-versions-of-r}

Exigimos que os pacotes da rOpenSci sejam testados nas versões mais recentes, porém também nas versões anteriores e de desenvolvimento do R, para garantir a compatibilidade retroativa e progressiva com o R básico.

Detalhes sobre como executar testes/verificações usando diferentes versões do R localmente podem ser encontrados na vinheta do R-hub ao executar [Verificações locais do Linux com Docker](https://r-hub.github.io/rhub/articles/local-debugging.html).

Você pode ajustar a implementação de testes com cada versão usando uma matriz de testes.

Se você desenvolver um pacote que dependa ou seja destinado ao Bioconductor, esta informação [biocthis](https://lcolladotor.github.io/biocthis/index.html) pode ser relevante.

#### Minimizando o tempo de compilação na CI {#minimizing-build-times-on-ci}

Você pode usar estas dicas para minimizar o tempo de compilação na CI:

> - Instalar os pacotes em um cache que possa ser reutilizado no processo de CI/CD quando necessário (_cache installation of packages_). O padrão faz isso [r-lib/actions workflows](https://github.com/r-lib/actions).

Author: If is not in the English version then we can remove it from here too.

> - Instalar os pacotes em um cache que possa ser reutilizado no processo de CI/CD quando necessário (_cache installation of packages_). O padrão faz isso [r-lib/actions workflows](https://github.com/r-lib/actions).

Author: ```suggestion
- Instalar os pacotes em um cache que possa ser reutilizado no processo de CI quando necessário (_cache installation of packages_). O padrão faz isso [r-lib/actions workflows](https://github.com/r-lib/actions).
```

#### Dependências do sistema {#sysdeps-ci}

Você pode achar a postagem de Hugo Gruson útil [Dependências do sistema em pacotes R e testes automáticos](https://blog.r-hub.io/2023/09/26/system-dependency/).

### Travis CI (Linux e Mac OSX) {#travis-ci-linux-and-mac-osx}

Recomendamos que você [afaste-se de Travis](https://ropensci.org/technotes/2020/11/19/moving-away-travis/).

### AppVeyor CI (Windows) {#app-veyor-ci-windows}

Para integração contínua no Windows, consulte [R + AppVeyor](https://github.com/krlmlr/r-appveyor). Configure-o usando `usethis::use_appveyor()`.

Aqui estão algumas dicas para você minimizar o tempo de compilação do AppVeyor:

> - Instalação em cache de pacotes. [Exemplo em um arquivo de configuração](https://github.com/r-lib/usethis/blob/2c52c06373849d52f78a26c5a0e080f518a2f825/inst/templates/appveyor.yml#L13). Ele já estará no arquivo de configuração se você configurar o AppVeyor CI usando `usethis::use_appveyor()`.

Author: @pedropark99 , @beatrizmilz, any ideas?

> - Instalação em cache de pacotes. [Exemplo em um arquivo de configuração](https://github.com/r-lib/usethis/blob/2c52c06373849d52f78a26c5a0e080f518a2f825/inst/templates/appveyor.yml#L13). Ele já estará no arquivo de configuração se você configurar o AppVeyor CI usando `usethis::use_appveyor()`.

Reviewer: I think this change makes more sense.

```suggestion
- Instale os seus pacotes em algum tipo de cache. [Exemplo de um arquivo de configuração](https://github.com/r-lib/usethis/blob/2c52c06373849d52f78a26c5a0e080f518a2f825/inst/templates/appveyor.yml#L13). Ele já estará no arquivo de configuração se você configurar o AppVeyor CI usando `usethis::use_appveyor()`.
```

> - Instalação em cache de pacotes. [Exemplo em um arquivo de configuração](https://github.com/r-lib/usethis/blob/2c52c06373849d52f78a26c5a0e080f518a2f825/inst/templates/appveyor.yml#L13). Ele já estará no arquivo de configuração se você configurar o AppVeyor CI usando `usethis::use_appveyor()`.

Author: Thank you!

### Circle CI (Linux e Mac OSX) {#circle-ci-linux-and-mac-osx}

[Circle CI](https://circleci.com/) é usado, por exemplo, pelo pacote [`bomrang`](https://github.com/ropensci/bomrang) da rOpenSci como serviço de integração contínua.

## Cobertura de testes {#coverage}

A integração contínua também deve incluir relatórios de cobertura de teste por meio de um serviço de teste, como o [Codecov](https://codecov.io/) ou [Coveralls](https://coveralls.io/).

Recomendamos que você use Codecov. Para ativar Codecov em seu repositório, execute `usethis::use_github_action("test-coverage")` para criar um arquivo `.github/workflows/test-coverage.yaml`. Você também precisa dar ao Codecov acesso ao seu repositório do GitHub, consulte [Guia de início rápido do Codecov](https://docs.codecov.com/docs/quick-start) para saber como configurar o acesso. Em seguida, adicione um distintivo de status do Codecov à parte superior do seu README.md, consulte [Distintivos de status do Codecov](https://docs.codecov.com/docs/status-badges).

> Atualmente, Codecov tem acesso a todos os repositórios do GitHub da rOpenSci por padrão. Quando o seu repositório for aceito e transferido para a rOpenSci, o acesso ao Codecov deverá ser transferido automaticamente. Você precisará atualizar o URL do distintivo para apontar ao repositório hospedado na rOpenSci.

Reviewer 2: ```suggestion
Se o seu repositório for transferido para a organização GitHub ropensci, o acesso ao Codecov deverá ser transferido automaticamente. Você precisará atualizar o URL do distintivo para apontar ao repositório hospedado na rOpenSci.
```

@beatrizmilz also a change from #864

Author: A hierarchy in GitHub, like you have users, you have organizations.  An organization can have many repositories within it.

rOpenSci has an organization on GitHub that has the repositories of all the packages that have gone through the peer review process.

> ---

Author: ```suggestion

```

> aliases: ci.html

Author: ```suggestion

```

> ---

Author: ```suggestion

```
